tests/core/test_basics.py

```diff
@@ -156,3 +190,13 @@ def expected_response_type() -> Type[DummyResponse]:
 
     await asyncio.sleep(0.01)
     assert isinstance(received, DummyRequest)
+
+
+@pytest.mark.asyncio
+async def test_wait_for_can_get_cancelled(endpoint: Endpoint) -> None:
+
+    with pytest.raises(asyncio.TimeoutError):
+        await asyncio.wait_for(endpoint.wait_for(DummyRequest), 0.1)
+    await asyncio.sleep(0.1)
```

Comment:
Can this get smaller?  `0.1` adds up quickly in tests runs.

Reply:
Yep, checked all sleeps and reduced times.